sublime-text-3/Dockerfile

```diff
@@ -1,16 +1,22 @@
-# VERSION:		0.1
+# VERSION:		0.2
 # DESCRIPTION:	Create sublime-text 3 container with its dependencies (https://www.sublimetext.com/3)
-# AUTHOR:		Christian Koep <christian.koep@fom-net.de>
+# AUTHORS:		Christian Koep <christian.koep@fom-net.de>, Chuck Knox <charles.m.knox@gmail.com>
 # USAGE:
 #	# Build sublime-text 3 image
 #	docker build -t sublime-text:3 .
 #
 #	# Run the container and mount the local settings and your code
-#	docker run -it \
-#		-v $HOME/.config/sublime-text-3/:/root/.config/sublime-text-3 \
-#		-v $HOME/development:/root/development \
-#		-v /tmp/.X11-unix:/tmp/.X11-unix \
-#		-e DISPLAY=$DISPLAY sublime-text:3
+#   # Your code must be under $HOME/Documents, you only need to change it here.
+#		docker run -d -it \
+#			-w $HOME/Documents \
+#			-v $HOME/.config/sublime-text-3:$HOME/.config/sublime-text-3 \
+#			-v $HOME/Documents:$HOME/Documents \
+#			-v /tmp/.X11-unix:/tmp/.X11-unix \
+#			-v $HOME/.local/share/recently-used.xbel:$HOME/.local/share/recently-used.xbel \
+#			-e DISPLAY=$DISPLAY \
+#			-e NEWUSER=$USER \
+#			-e LANG=en_US.UTF-8 \
+#			sublime-text:3 /run.sh
```

Comment:
You are correct - I goofed up that one. It works with and without it. I'll close this pull request and submit another one with the corrected documentation.